### DISCUSSION TITLE
Question About Device Powered On/Off and Device Groups

### DISCUSSION THREAD

**Discussion by JDogg016** [music-assistant/support#3117] [Q&A] [answered]
What is the purpose for each of the music assistant speakers to be shown as having power on, or power off?

It seems that when you begin playing a speaker that is off, it turns on. However, when you stop playing that speaker, the power does not turn off. Why?

Also, a lot of speakers seem to be powered off when music assistant starts, and I cannot find a way to determine what would cause a speaker to be powered on or off when music assistant starts. Is there some sort of documentation on this or reason for this? I did see where a speaker group would start to play. It turned on, but that seems different than what I'm asking.

What I do notice, however, is that when you have a group of speakers and you power them on this removes the ability to individually control the playback of speakers in that group unless you either make the group a dynamic group or power the group off. Can someone please confirm this? 

The reason I ask all of these questions is, I am trying to determine whether a group of speakers should be a temporary group that I create using an automation, a permanent group that I would control using power on and power off or a dynamic group (which seems like something that would break other automations because speakers could join and/or leave a dynamic group). The particular group of speakers that I am interested in creating would control all of the speakers on the downstairs in my house so that if I want to play an announcement, I can simply play it to downstairs speakers.

**Comment by OzGav** [comment, marked as answer]
https://music-assistant.io/ui/#player-list
https://music-assistant.io/ui/#grouping-players

https://music-assistant.io/faq/groups

Most if not all of your questions are answered in the above 

edit: little bit more here (second last item) https://music-assistant.io/faq/troubleshooting/#ma-is-interfering-with-my-spotify-connect-or-other-streaming-app
